examples/ucnf-kiknos/README.md

```diff
@@ -0,0 +1,191 @@
+- `VPP_AGENT` - Parent image for the NSE - (Default: `ciscolabs/kiknos:latest`)
+- `FORWARDING_PLANE` - Set a default forwarding plane - (Default: `vpp`)
+- `NETWORK_SERVICE` - Set a default network service for Example clients - (Default: `hello-world`)
+- `BUILD_IMAGE` - Set whether to build the image or not - (Default: `true`)
+- `PROVISION_MODE` - Set the mode to provision the built image. Default "push"
+    one of "push" or "kind-load"
+    not relevant if $BUILD_IMAGE is not true - (Default: `push`)
+- `DEPLOY_ISTIO` - Set whether to deploy istio gateway or not - (Default: `true`)
+- `AWS` - Create aws cluster - (Default: `false`)
+- `SUBNET_IP` - IP for the remote ASA subnet (without the mask, ex: 192.168.254.0) - (Default: `192.168.254.0`)
+
+## Check connectivity between workloads (Scenario 1)
+In order to check the connectivity between worker pods run the following make target:
+```bash
+./examples/ucnf-kiknos/scripts/test_vpn_conn.sh --cluster1=kiknos-demo-1 --cluster2=kiknos-demo-2 
+```
+This will attempt to perform `curl` commands from the workers in the VPN Gateway cluster to the workers in the VPN client cluster directly.
+
+Output:
+```bash
+CLUSTER1=kiknos-demo-1 CLUSTER2=kiknos-demo-2 /home/mihai/go/src/github.com/networkservicemesh/examples/examples/ucnf-kiknos/scripts/test_vpn_conn.sh
+Detected pod with nsm interface ip: 172.31.22.5
+Detected pod with nsm interface ip: 172.31.22.1
+Hello version: v1, instance: helloworld-ucnf-client-7bd94648d-d2gbh 
+Hello version: v1, instance: helloworld-ucnf-client-7bd94648d-nksbm
+Hello version: v1, instance: helloworld-ucnf-client-7bd94648d-d2gbh
+Hello version: v1, instance: helloworld-ucnf-client-7bd94648d-nksbm
+```
+
+## Check connectivity to workloads through the Istio gateway (Scenario 2)
+In order to check the connectivity to workloads through the Istio gateway run the following make target:
+```bash
+./examples/ucnf-kiknos/scripts/test_istio_vpn_conn.sh --cluster1=kiknos-demo-1 --cluster2=kiknos-demo-2
+```
+This will attempt to perform `curl` commands from the workers in the VPN Gateway cluster to the Istio ingress gateway.
+This allows a user to connect to different services by keeping the same IP address for each call and differentiating 
+between workloads by specifying different ports or URL paths. 
+
+In our current example, we deploy 2 services, and expose each service twice. In all cases, the request goes through the
+IP address of the Ingress gateway that was supplied by the NSE.
+
+Output:
+```bash
+CLUSTER1=kiknos-demo-1 CLUSTER2=kiknos-demo-2 /home/mihai/go/src/github.com/networkservicemesh/examples/examples/ucnf-kiknos/scripts/test_istio_vpn_conn.sh
+Detected pod with nsm interface ip: 172.31.22.9
+------------------------- Source pod/helloworld-ucnf-client-7bd94648d-2t7zj -------------------------
+Connecting to: http://172.31.22.9/hello
+Hello version: v1, instance: icmp-responder-v1-88bcb54bf-vt8jv
```

Comment:
I thought we were going to remove icmp-responder from the naming.  It looks like we have done that a few places but not everywhere.

Reply:
changed all icmp-responder occurrences